src/build/env.py

```diff
@@ -139,6 +155,12 @@ def python_executable(self) -> str:
         """The python executable of the isolated build environment."""
         return self._python_executable
 
+    def _pip_args(self, *, isolate: bool = False) -> list[str]:
```

Comment:
This doesn't appear to be ever be called without `isolate` being true.

Reply:
Okay, dropped the arg.